TestStack.Dossier/TestDataBuilder.cs

```diff
@@ -71,10 +72,22 @@ public TObject Build()
         }
 
         /// <summary>
-        /// Build the actual object - override this and call the constructor and any other methods.
+        /// Build the actual object - you can optionally override this and call the constructor and any other methods.
         /// </summary>
         /// <returns>The built object</returns>
-        protected abstract TObject BuildObject();
+        protected virtual TObject BuildObject()
+        {
+            var model = Any.Fixture.Create<TObject>();
+
+            var properties = Reflector.GetSettablePropertiesFor<TObject>();
```

Comment:
I think we should make it as easy as possible for user by doing as much as we can and let them choose how they use it.